README.md

```diff
@@ -145,6 +143,20 @@ app.listen(1337, function() {
 
 For a full list of available options, run `parse-server --help`.
 
+## Logging
+
+Parse Server will, by default, will log:
+* to the console
+* daily rotating files as new line delimited JSON
+
+Logs are also be viewable in Parse Dashboard but it only displays the `messages` field of each log entry. For example, with VERBOSE set this will exclude `origin` on each request.
```

Comment:
This paragraph is probably unnecessary. Could be shortened to "Logs are also viewable in Parse Dashboard."